### ISSUE TITLE
is it possible to use multi GPUs to speed up the inference?

### ISSUE THREAD

**Issue by youchangxin** [alembics/disco-diffusion#115]

**Comment by MoYuFly** [comment]
Same question, request reply

**Comment by aletts** [comment]
Currently, Disco Diffusion doesn't support this. In theory there are ways to do this.. but it hasn't been implemented. Its history of being primarily used via Google Colab (and by relatively few users running locally, often with a single consumer Nvidia GPU) made implementation of it low priority.

**Comment by MoYuFly** [comment]
“Allow 3D animation to run on GPU's other than GPU0”，Can I use multiple GPUs, or can I specify other GPUs？

**Comment by aletts** [comment]
It's easy to make it run on a different GPU, an it's something that I do sometimes. You can search for the "cuda:0" string in disco.py and change it to your desired GPU index. Then you can e.g. run several instances of Disco Diffusion on the same system to use all the system's GPUs in parallel (working on separate jobs).

**Comment by MoYuFly** [comment]
It's inconvenient to access foreign Internet. I hope to realize this function as soon as possible. Thank you very much！

**Comment by youchangxin** [comment]
> It's easy to make it run on a different GPU, an it's something that I do sometimes. You can search for the "cuda:0" string in disco.py and change it to your desired GPU index. Then you can e.g. run several instances of Disco Diffusion on the same system to use all the system's GPUs in parallel (working on separate jobs).

Thank you for your suggestion, but it may not fit my job well. I am struggling with multiprocess to speed up the diffusion step.
